camelot/view/action_steps/form_view.py

```diff
@@ -154,6 +151,24 @@ def gui_run(cls, gui_context_name, serialized_step):
                 ))
                 show_top_level(formview, gui_context_name, step['form_state'])
 
+@dataclass
+class HighlightForm(ActionStep, DataclassSerializable):
+
+    tab: Optional[str] = None
+    table_label: Optional[str] = None
+    table_row: Optional[int] = None
+    table_column: Optional[str] = None
+
+    #action_cls: Optional[?] = None
```

Comment:
idd questionable, die cls zaken zijn eigenlijk nt gekend aan de gui kant, dus het heeft ook geen zin die door te sturen.

die moeten denk ik eerder server side w omgezet van een 'action_cls' naar een 'action_route', die dan wel door de gui gekend is